### ISSUE TITLE
3.15.0 Exponea Init error - java.lang.NoSuchMethodError: No virtual method add

### ISSUE THREAD

**Issue by fabiawrc** [exponea/exponea-android-sdk#75]
Hi, after updating Android Exponea sdk from `3.14.0` to `3.15.0` we are getting an error during initialization. 
It looks like it is somehow related to the addition of the coil library for image support 🤔 . 

```
Exponea Safe Mode wrapper caught unhandled error (Ask Gemini)
                 java.lang.NoSuchMethodError: No virtual method add(Lcoil/decode/Decoder;)Lcoil/ComponentRegistry$Builder; in class Lcoil/ComponentRegistry$Builder; or its super classes (declaration of 'coil.ComponentRegistry$Builder' appears in /data/app/~~Ain7B2yNnnmgg0T4msSfNw==/cz.csob.sp.acc--NbPpjndXrXRuUl2fGTcGQ==/base.apk!classes19.dex)
                 	at com.exponea.sdk.repository.SimpleDrawableCache.<init>(SimpleDrawableCache.kt:27)
                 	at com.exponea.sdk.repository.InAppMessageBitmapCacheImpl.<init>(InAppMessageBitmapCacheImpl.kt:7)
                 	at com.exponea.sdk.ExponeaComponent.<init>(ExponeaComponent.kt:141)
                 	at com.exponea.sdk.Exponea.initializeSdk(Exponea.kt:756)
                 	at com.exponea.sdk.Exponea.init(Exponea.kt:384)
```

**Comment by adam1929** [comment]
Hi @fabiawrc thank you for report. Yes SDK and your app are using different Coil version that are incompatible with each other. Please report this issue with your CSM through support channel to investigate it more deeply and bring next steps more efficiently.
Thank you

**Comment by xcibik00** [comment]
Hi @fabiawrc - its already resolved in newest 3.16.0 version

**Comment by fabiawrc** [comment]
@xcibik00 thank you!
